index.html

```diff
@@ -41,47 +41,47 @@ <h4 class="mb-3">Who are you?</h4>
                 <form>
                   <div class="row">
                     <div class="col-md-6 mb-36">
-                      <label for="jobTitle">First Name</label>
-                      <input type="text" class="form-control" id="firstName" placeholder="..." value="">
+                      <label for="signUpFirstName">First Name</label>
+                      <input type="text" class="form-control" id="signUpFirstName" placeholder="..." value="">
                     </div>
   
                     <div class="col-md-6 mb-6">
-                      <label for="companyName">Last Name</label>
-                      <input type="text" class="form-control" id="lastName" placeholder="..." value="">
+                      <label for="signUpLastName">Last Name</label>
+                      <input type="text" class="form-control" id="signUpLastName" placeholder="..." value="">
                     </div>
                     <div class="col-md-6 mb-6">
-                        <label for="phoneNumber">Phone Number</label>
-                        <input type="text" class="form-control" id="phoneNumber" placeholder="..." value="">
+                        <label for="signUpPhoneNumber">Phone Number</label>
+                        <input type="text" class="form-control" id="signUpPhoneNumber" placeholder="..." value="">
                     </div>
                     <div class="col-md-6 mb-6">
-                        <label for="email">E-mail</label>
-                        <input type="text" class="form-control" id="email" placeholder="..." value="">
+                        <label for="signUpEmail">E-mail</label>
+                        <input type="text" class="form-control" id="signUpEmail" placeholder="..." value="">
                     </div>
                   </div>
   
                   <div class="mb-3">
-                    <label for="address">Address</label>
-                    <input type="text" class="form-control" id="workAddress" placeholder="1234 Main St" required="">
-                    <div class="invalid-feedback">Please enter the address.
+                    <label for="signUpAddress">Address</label>
+                    <input type="text" class="form-control" id="signUpAddress" placeholder="1234 Main St" required="">
+                    <div class="invalid-feedback">Please enter the asignUpAdress.
```

Comment:
Shouldn't the printed text here say something along the lines of "Please enter an address"?

Reply:
yeah that was a typo on my part. Line 65 should say "Please enter your address"
```suggestion
                    <div class="invalid-feedback">Please enter your address
```